synapse/tests/test_daemon.py

```diff
@@ -294,3 +311,20 @@ def test_daemon_cells(self):
         # ensure dmon cell processes are fini'd
         for celldir, proc in dmon.cellprocs.items():
             self.false(proc.is_alive())
+
+    def test_daemon_aware(self):
+
+        woke = Woke()
+        self.eq(woke.hehe(2), 40)
+        self.eq(woke.echo('echo'), 'echo')
+
+        with s_daemon.Daemon() as dmon:
+            link = dmon.listen('tcp://127.0.0.1:0/')
```

Comment:
As above, this will break when the OSX PR is merged

Reply:
we can bind to port 0 (and get a random port back) on osx; we just cannot connect to port 0.

Reply:
ok